actions/src/project_action.py

```diff
@@ -80,3 +80,38 @@ def project_create(
             cloud_account=cloud_account, project_details=details
         )
         return bool(project), project
+
+    def project_update(
+        self,
+        cloud_account: str,
+        project_identifier: str,
+        name: str,
+        email: str,
+        description: str,
+        is_enabled: str,
+    ) -> Tuple[bool, Optional[Project]]:
+        """
+        Find and update a given project's properties.
+        :param cloud_account: The account from the clouds configuration to use
+        :param project_identifier: Name or Openstack ID of the project to update
+        :param name: Name of the project
+        :param email: Contact email of the project
+        :param description: Description of the project
+        :param is_enabled: Enable or disable the project (takes values of unchanged, true or false)
+        :return: status, optional project
+        """
+        is_enabled_value = None
+        if is_enabled == "true":
```

Comment:
I'd do this using casefold, since I saw Megan fighting a bug yesterday where it randomly changed case on something that should never change case:

```
if is_enabled.casefold() == "true".casefold():
```